app/assets/stylesheets/templates/_home.scss

```diff
@@ -99,7 +99,7 @@
         }
       }
 
-      &.stimulus {
+      &.diy {
```

Comment:
Stimulus page / class is no longer in use

Reply:
Stellar